-code-checks

Logs: https://cmssdt.cern.ch/SDT/code-checks/cms-sw-PR-40513/33803

ERROR: Build errors found during clang-tidy run.
```
DQM/PixelLumi/plugins/PixelLumiDQM.cc:429:12: error: expected '(' for function-style cast or type construction [clang-diagnostic-error]
    double etotal = useInnerBarrelLayer 
           ^
Suppressed 1787 warnings (1786 in non-user code, 1 with check filters).
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:129: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/DQM/BeamMonitor/plugins/FakeBeamMonitor.cc
+++ b/DQM/BeamMonitor/plugins/FakeBeamMonitor.cc
@@ -854,7 +854,6 @@ void FakeBeamMonitor::FitAndFill(const LuminosityBlock& lumiSeg, int& lastlumi, 
 
   int MaxPVs = 0;
   int countEvtLastNLS_ = 0;
-  int countTotPV_ = 0;
 
   std::map<int, std::vector<int> >::iterator mnpv = mapNPV.begin();
   std::map<int, std::vector<float> >::iterator mpv2 = mapPVy.begin();
@@ -878,7 +877,6 @@ void FakeBeamMonitor::FitAndFill(const LuminosityBlock& lumiSeg, int& lastlumi, 
       if ((*mnpvs > 0) && (resetHistoFlag_))
         h_nVtx_st->Fill((*mnpvs) * (1.0));
       countEvtLastNLS_++;
-      countTotPV_ += (*mnpvs);
       if ((*mnpvs) > MaxPVs)
         MaxPVs = (*mnpvs);
     }  //loop over second of mapNPV
@@ -891,26 +889,6 @@ void FakeBeamMonitor::FitAndFill(const LuminosityBlock& lumiSeg, int& lastlumi, 
 
   std::vector<float> DipPVInfo_;
   DipPVInfo_.clear();
-  //
-  //  if (countTotPV_ != 0) {
-  //    DipPVInfo_.push_back((float)countEvtLastNLS_);
-  //    DipPVInfo_.push_back(h_nVtx_st->getMean());
-  //    DipPVInfo_.push_back(h_nVtx_st->getMeanError());
-  //    DipPVInfo_.push_back(h_nVtx_st->getRMS());
-  //    DipPVInfo_.push_back(h_nVtx_st->getRMSError());
-  //    DipPVInfo_.push_back((float)MaxPVs);
-  //    DipPVInfo_.push_back((float)countTotPV_);
-  //    MaxPVs = 0;
-  //  } else {
-  //    for (size_t i = 0; i < 7; i++) {
-  //      if (i > 0) {
-  //        DipPVInfo_.push_back(0.);
-  //      } else {
-  //        DipPVInfo_.push_back((float)countEvtLastNLS_);
-  //      }
-  //    }
-  //  }
-  //  theBeamFitter->SetPVInfo(DipPVInfo_);
   DipPVInfo_.push_back(rndm_->Gaus(1000., 100.));  // Events used
   DipPVInfo_.push_back(rndm_->Gaus(100., 10.));    // Mean PV
   DipPVInfo_.push_back(rndm_->Gaus(10., 5.));      // Mean PV err

--- a/DQM/CastorMonitor/src/CastorDigiMonitor.cc
+++ b/DQM/CastorMonitor/src/CastorDigiMonitor.cc
@@ -224,7 +224,6 @@ void CastorDigiMonitor::processEvent(edm::Event const &event,
 
     int capid1 = digi.sample(0).capid();
     hdigisize->Fill(digi.size());
-    double sum = 0.;
     int err = 0, err2 = 0;
     for (int i = 0; i < digi.size(); i++) {
       int capid = digi.sample(i).capid();
@@ -239,7 +238,6 @@ void CastorDigiMonitor::processEvent(edm::Event const &event,
       h2QmeantsvsCh->Fill(ind, i, LedMonAdc2fc[rawd]);
       float q = LedMonAdc2fc[rawd];
       Ecell[module][sector] = q;
-      sum += q;  //     sum += LedMonAdc2fc[rawd];
       QrmsTS[ind][i] += (q * q);
       QmeanTS[ind][i] += q;
       if (err != 0 && fVerbosity > 0)

--- a/DQM/CastorMonitor/src/CastorRecHitMonitor.cc
+++ b/DQM/CastorMonitor/src/CastorRecHitMonitor.cc
@@ -182,7 +182,6 @@ void CastorRecHitMonitor::processEvent(const CastorRecHitCollection &castorHits)
     h2RHentriesMap->Fill(module - 1, sector - 1);
   }  // end for(CASTORiter=castorHits.begin(); CASTORiter!= ...
 
-  double etot = 0.;
   for (int phi = 0; phi < 16; phi++) {
     double es = 0.;
     for (int z = 0; z < 14; z++) {
@@ -197,7 +196,6 @@ void CastorRecHitMonitor::processEvent(const CastorRecHitCollection &castorHits)
       es += rh;
     }
     h2RHvsSec->Fill(phi, es);
-    etot += es;
   }  // end for(int phi=0;
 
   if (fVerbosity > 0)

--- a/DQM/L1TMonitorClient/src/L1TOccupancyClient.cc
+++ b/DQM/L1TMonitorClient/src/L1TOccupancyClient.cc
@@ -680,7 +680,6 @@ void L1TOccupancyClient::printDeadChannels(const vector<pair<int, double> >& iDe
   }
 
   int x, y, z;
-  float chi2 = 0.0;
 
   // put all bad (value=1) and masked (value=-1) cells in histo
   for (std::vector<pair<int, double> >::const_iterator it = iDeadChannels.begin(); it != iDeadChannels.end(); it++) {
@@ -699,13 +698,6 @@ void L1TOccupancyClient::printDeadChannels(const vector<pair<int, double> >& iDe
       }
     }
   }
-
-  // FIXME: Is this needed?
-  for (std::vector<pair<int, double> >::const_iterator it = statDev.begin(); it != statDev.end(); it++) {
-    double dev = (*it).second;
-    chi2 += dev;
-  }
-  //put total chi2 in float
 
   if (verbose_) {
     cout << "total number of suspect channels: " << (iDeadChannels.size() - (hservice_->getNBinsMasked(iTestName)))

--- a/DQM/Physics/src/EwkTauDQM.cc
+++ b/DQM/Physics/src/EwkTauDQM.cc
@@ -248,16 +248,13 @@ void EwkElecTauHistManager::fillHistograms(const edm::Event& evt, const edm::Eve
 
   double theElectronTrackIsoPt = 1.e+3;
   double theElectronEcalIsoPt = 1.e+3;
-  double theElectronHcalIsoPt = 1.e+3;
   if (theElectron) {
     theElectronTrackIsoPt = theElectron->dr03TkSumPt();
     theElectronEcalIsoPt = theElectron->dr03EcalRecHitSumEt();
-    theElectronHcalIsoPt = theElectron->dr03HcalTowerSumEt();
 
     if (electronIsoMode_ == kRelativeIso && theElectron->pt() > 0.) {
       theElectronTrackIsoPt /= theElectron->pt();
       theElectronEcalIsoPt /= theElectron->pt();
-      theElectronHcalIsoPt /= theElectron->pt();
     }
   }
 

--- a/DQM/PixelLumi/plugins/PixelLumiDQM.cc
+++ b/DQM/PixelLumi/plugins/PixelLumiDQM.cc
@@ -411,7 +411,6 @@ void PixelLumiDQM::endLuminosityBlock(edm::LuminosityBlock const &lumiBlock, edm
   // Obtain bunch-by-bunch cluster counts and compute totals for lumi
   // calculation.
   double totalcounts = 0.0;
-  double etotalcounts = 0.0;
   double totalevents = 0.0;
   double lumi_factor_per_bx = 0.0;
   if (useInnerBarrelLayer)
@@ -426,11 +425,10 @@ void PixelLumiDQM::endLuminosityBlock(edm::LuminosityBlock const &lumiBlock, edm
     if (useInnerBarrelLayer)
       total += (*it).second.numB.at(0);
     totalcounts += total;
-    double etotal = (*it).second.dnumB.at(1) + (*it).second.dnumB.at(2) + (*it).second.dnumFP.at(0) +
-                    (*it).second.dnumFP.at(1) + (*it).second.dnumFM.at(0) + (*it).second.dnumFM.at(1);
-    if (useInnerBarrelLayer)
-      etotal = (*it).second.dnumB.at(0);
-    etotalcounts += etotal;
+    double etotal = useInnerBarrelLayer
+                        ? (*it).second.dnumB.at(0)
+                        : (*it).second.dnumB.at(1) + (*it).second.dnumB.at(2) + (*it).second.dnumFP.at(0) +
+                              (*it).second.dnumFP.at(1) + (*it).second.dnumFM.at(0) + (*it).second.dnumFM.at(1);
     etotal = sqrt(etotal);
 
     fHistClusterCountByBxLastLumi->setBinContent((*it).first, total);

--- a/DQMOffline/CalibTracker/plugins/SiStripPopConBadComponentsDQM.cc
+++ b/DQMOffline/CalibTracker/plugins/SiStripPopConBadComponentsDQM.cc
@@ -92,8 +92,6 @@ void SiStripPopConBadComponentsHandlerFromDQM::dqmEndJob(DQMStore::IBooker&, DQM
 
   const std::vector<std::string> subdet_folder = {"TIB", "TOB", "TEC/side_1", "TEC/side_2", "TID/side_1", "TID/side_2"};
 
-  int nDetsTotal = 0;
-  int nDetsWithErrorTotal = 0;
   for (const auto& im : subdet_folder) {
     const std::string dname = mechanicalview_dir + "/" + im;
     getter.cd();
@@ -103,14 +101,11 @@ void SiStripPopConBadComponentsHandlerFromDQM::dqmEndJob(DQMStore::IBooker&, DQM
 
     std::vector<std::string> module_folders;
     getModuleFolderList(getter, dname, module_folders);
-    int nDets = module_folders.size();
 
-    int nDetsWithError = 0;
     const std::string bad_module_folder = dname + "/" + "BadModuleList";
     getter.cd();
     if (getter.dirExists(bad_module_folder)) {
       for (const MonitorElement* me : getter.getContents(bad_module_folder)) {
-        nDetsWithError++;
         std::cout << me->getName() << " " << me->getIntValue() << std::endl;
         uint32_t detId = std::stoul(me->getName());
         short flag = me->getIntValue();
@@ -140,8 +135,6 @@ void SiStripPopConBadComponentsHandlerFromDQM::dqmEndJob(DQMStore::IBooker&, DQM
         }
       }
     }
-    nDetsTotal += nDets;
-    nDetsWithErrorTotal += nDetsWithError;
   }
   getter.cd();
 }

--- a/DQMOffline/JetMET/src/ECALRecHitAnalyzer.cc
+++ b/DQMOffline/JetMET/src/ECALRecHitAnalyzer.cc
@@ -267,7 +267,6 @@ void ECALRecHitAnalyzer::FillGeometry(const edm::EventSetup& iSetup) {
   }
 
   //-------Set the cell size for each (ieta, iphi) bin-------//
-  double currentLowEdge_eta = 0;
   //double currentHighEdge_eta = 0;
   for (int ieta = 1; ieta <= 85; ieta++) {
     int ieta_ = 86 + ieta;
@@ -284,7 +283,6 @@ void ECALRecHitAnalyzer::FillGeometry(const edm::EventSetup& iSetup) {
     double deta = fabs(eta - etam1);
     double dphi = fabs(hEB_ieta_iphi_phiMap->getBinContent(ieta_, 1) - hEB_ieta_iphi_phiMap->getBinContent(ieta_, 2));
 
-    currentLowEdge_eta += deta;
     hEB_ieta_detaMap->setBinContent(ieta_, deta);      // positive rings
     hEB_ieta_dphiMap->setBinContent(ieta_, dphi);      // positive rings
     hEB_ieta_detaMap->setBinContent(86 - ieta, deta);  // negative rings

--- a/DQMOffline/L1Trigger/src/L1TPhase2OuterTrackerTkMET.cc
+++ b/DQMOffline/L1Trigger/src/L1TPhase2OuterTrackerTkMET.cc
@@ -85,10 +85,8 @@ void L1TPhase2OuterTrackerTkMET::analyze(const edm::Event& iEvent, const edm::Ev
   }
   float sumPx = 0;
   float sumPy = 0;
-  float etTot = 0;
   double sumPx_PU = 0;
   double sumPy_PU = 0;
-  double etTot_PU = 0;
   int nTracks_counter = 0;
 
   float zVTX = L1VertexHandle->begin()->z0();
@@ -155,11 +153,9 @@ void L1TPhase2OuterTrackerTkMET::analyze(const edm::Event& iEvent, const edm::Ev
 
       sumPx += pt * cos(phi);
       sumPy += pt * sin(phi);
-      etTot += pt;
     } else {  // PU sums
       sumPx_PU += pt * cos(phi);
       sumPy_PU += pt * sin(phi);
-      etTot_PU += pt;
     }
   }  // end loop over tracks
 

--- a/DQMServices/Components/plugins/DQMStoreStats.cc
+++ b/DQMServices/Components/plugins/DQMStoreStats.cc
@@ -749,7 +749,7 @@ int DQMStoreStats::calcstats(int mode = DQMStoreStats::considerAllME) {
 
   // dump mem/bin table
 
-  unsigned int overallNHistograms = 0, overallNBins = 0, overallNEmptyBins = 0, overallNBytes = 0;
+  unsigned int overallNHistograms = 0, overallNBins = 0, overallNBytes = 0;
 
   std::cout << std::endl;
   std::cout << "------------------------------------------------------------------------------------------"
@@ -815,7 +815,6 @@ int DQMStoreStats::calcstats(int mode = DQMStoreStats::considerAllME) {
 
     overallNHistograms += nHistograms;
     overallNBins += nBins;
-    overallNEmptyBins += nEmptyBins;
     overallNBytes += nBytes;
 
     // display totals
@@ -872,7 +871,6 @@ int DQMStoreStats::calcstats(int mode = DQMStoreStats::considerAllME) {
       return 0;
     // Prepare appropriate map to store FWJR output.
     std::map<std::string, std::string> jrInfo;
-    unsigned int overallNHistograms = 0, overallNBins = 0, overallNBytes = 0;
 
     jrInfo["Source"] = "DQMServices/Components";
     jrInfo["FileClass"] = "DQMStoreStats";
@@ -900,9 +898,6 @@ int DQMStoreStats::calcstats(int mode = DQMStoreStats::considerAllME) {
         nEmptyBins += it1->totalEmptyBins_;
         nBytes += it1->totalMemory_;
       }
-      overallNHistograms += nHistograms;
-      overallNBins += nBins;
-      overallNBytes += nBytes;
       std::stringstream iss("");
       iss << nHistograms;
       jrInfo[it0->subsystemName_ + std::string("_h")] = iss.str();

--- a/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_DoubleEle_Hadronic.cc
+++ b/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_DoubleEle_Hadronic.cc
@@ -194,7 +194,6 @@ void SUSY_HLT_DoubleEle_Hadronic::analyze(edm::Event const &e, edm::EventSetup c
       offlineCounter++;
     }
 
-    float caloHT = 0.0;
     float pfHT = 0.0;
     for (reco::PFJetCollection::const_iterator i_pfjet = pfJetCollection->begin(); i_pfjet != pfJetCollection->end();
          ++i_pfjet) {
@@ -211,7 +210,6 @@ void SUSY_HLT_DoubleEle_Hadronic::analyze(edm::Event const &e, edm::EventSetup c
         continue;
       if (fabs(i_calojet->eta()) > etaThrJet_)
         continue;
-      caloHT += i_calojet->pt();
     }
 
     if (hasFiredAuxiliaryForElectronLeg && ElectronCollection->size() > 1) {

--- a/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_DoubleMuon_Hadronic.cc
+++ b/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_DoubleMuon_Hadronic.cc
@@ -184,7 +184,6 @@ void SUSY_HLT_DoubleMuon_Hadronic::analyze(edm::Event const &e, edm::EventSetup 
       offlineCounter++;
     }
 
-    float caloHT = 0.0;
     float pfHT = 0.0;
     for (reco::PFJetCollection::const_iterator i_pfjet = pfJetCollection->begin(); i_pfjet != pfJetCollection->end();
          ++i_pfjet) {
@@ -201,7 +200,6 @@ void SUSY_HLT_DoubleMuon_Hadronic::analyze(edm::Event const &e, edm::EventSetup 
         continue;
       if (fabs(i_calojet->eta()) > etaThrJet_)
         continue;
-      caloHT += i_calojet->pt();
     }
 
     if (hasFiredAuxiliaryForMuonLeg && MuonCollection->size() > 1) {

--- a/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_MuEle_Hadronic.cc
+++ b/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_MuEle_Hadronic.cc
@@ -236,7 +236,6 @@ void SUSY_HLT_MuEle_Hadronic::analyze(edm::Event const &e, edm::EventSetup const
       offlineCounter++;
     }
 
-    float caloHT = 0.0;
     float pfHT = 0.0;
     for (reco::PFJetCollection::const_iterator i_pfjet = pfJetCollection->begin(); i_pfjet != pfJetCollection->end();
          ++i_pfjet) {
@@ -253,7 +252,6 @@ void SUSY_HLT_MuEle_Hadronic::analyze(edm::Event const &e, edm::EventSetup const
         continue;
       if (fabs(i_calojet->eta()) > etaThrJet_)
         continue;
-      caloHT += i_calojet->pt();
     }
 
     if (hasFiredAuxiliaryForMuEleLeg && !MuonCollection->empty() && !ElectronCollection->empty()) {

--- a/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_Muon_Hadronic.cc
+++ b/HLTriggerOffline/SUSYBSM/src/SUSY_HLT_Muon_Hadronic.cc
@@ -211,7 +211,6 @@ void SUSY_HLT_Muon_Hadronic::analyze(edm::Event const &e, edm::EventSetup const 
       }
       offlineCounter++;
     }
-    float caloHT = 0.0;
     float pfHT = 0.0;
     for (reco::PFJetCollection::const_iterator i_pfjet = pfJetCollection->begin(); i_pfjet != pfJetCollection->end();
          ++i_pfjet) {
@@ -228,7 +227,6 @@ void SUSY_HLT_Muon_Hadronic::analyze(edm::Event const &e, edm::EventSetup const 
         continue;
       if (fabs(i_calojet->eta()) > etaThrJet_)
         continue;
-      caloHT += i_calojet->pt();
     }
     // Need to apply the MET offline cut to be in the MET plateau
     if ((hasFiredAuxiliaryForMuonLeg || !e.isRealData()) && !offlineMuons.empty() &&

--- a/Validation/EcalDigis/plugins/EcalDigisValidation.cc
+++ b/Validation/EcalDigis/plugins/EcalDigisValidation.cc
@@ -236,7 +236,6 @@ void EcalDigisValidation::analyze(edm::Event const& e, edm::EventSetup const& c)
 
       double Emax = 0.;
       int Pmax = 0;
-      double pedestalPreSample = 0.;
       double pedestalPreSampleAnalog = 0.;
 
       for (int sample = 0; sample < nrSamples; ++sample) {
@@ -256,14 +255,12 @@ void EcalDigisValidation::analyze(edm::Event const& e, edm::EventSetup const& c)
           Pmax = sample;
         }
         if (sample < 3) {
-          pedestalPreSample += ebADCCounts[sample];
           pedestalPreSampleAnalog += ebADCCounts[sample] * gainConv_[(int)ebADCGains[sample]] * barrelADCtoGeV_;
         }
         LogDebug("DigiInfo") << "EB sample " << sample << " ADC counts = " << ebADCCounts[sample]
                              << " Gain Id = " << ebADCGains[sample] << " Analog eq = " << ebAnalogSignal[sample];
       }
 
-      pedestalPreSample /= 3.;
       pedestalPreSampleAnalog /= 3.;
       double Erec = Emax - pedestalPreSampleAnalog * gainConv_[(int)ebADCGains[Pmax]];
 
@@ -322,7 +319,6 @@ void EcalDigisValidation::analyze(edm::Event const& e, edm::EventSetup const& c)
 
       double Emax = 0.;
       int Pmax = 0;
-      double pedestalPreSample = 0.;
       double pedestalPreSampleAnalog = 0.;
 
       for (int sample = 0; sample < nrSamples; ++sample) {
@@ -342,13 +338,11 @@ void EcalDigisValidation::analyze(edm::Event const& e, edm::EventSetup const& c)
           Pmax = sample;
         }
         if (sample < 3) {
-          pedestalPreSample += eeADCCounts[sample];
           pedestalPreSampleAnalog += eeADCCounts[sample] * gainConv_[(int)eeADCGains[sample]] * endcapADCtoGeV_;
         }
         LogDebug("DigiInfo") << "EE sample " << sample << " ADC counts = " << eeADCCounts[sample]
                              << " Gain Id = " << eeADCGains[sample] << " Analog eq = " << eeAnalogSignal[sample];
       }
-      pedestalPreSample /= 3.;
       pedestalPreSampleAnalog /= 3.;
       double Erec = Emax - pedestalPreSampleAnalog * gainConv_[(int)eeADCGains[Pmax]];
 

--- a/Validation/GlobalDigis/src/GlobalDigisAnalyzer.cc
+++ b/Validation/GlobalDigis/src/GlobalDigisAnalyzer.cc
@@ -570,7 +570,6 @@ void GlobalDigisAnalyzer::fillECal(const edm::Event &iEvent, const edm::EventSet
 
         double Emax = 0;
         int Pmax = 0;
-        double pedestalPreSample = 0.;
         double pedestalPreSampleAnalog = 0.;
 
         for (int sample = 0; sample < nrSamples; ++sample) {
@@ -590,12 +589,10 @@ void GlobalDigisAnalyzer::fillECal(const edm::Event &iEvent, const edm::EventSet
             Pmax = sample;
           }
           if (sample < 3) {
-            pedestalPreSample += ebADCCounts[sample];
             pedestalPreSampleAnalog +=
                 ebADCCounts[sample] * ECalgainConv_[(int)ebADCGains[sample]] * ECalbarrelADCtoGeV_;
           }
         }
-        pedestalPreSample /= 3.;
         pedestalPreSampleAnalog /= 3.;
 
         // calculate pedestal subtracted digi energy in the crystal
@@ -682,7 +679,6 @@ void GlobalDigisAnalyzer::fillECal(const edm::Event &iEvent, const edm::EventSet
 
         double Emax = 0;
         int Pmax = 0;
-        double pedestalPreSample = 0.;
         double pedestalPreSampleAnalog = 0.;
 
         for (int sample = 0; sample < nrSamples; ++sample) {
@@ -703,12 +699,10 @@ void GlobalDigisAnalyzer::fillECal(const edm::Event &iEvent, const edm::EventSet
             Pmax = sample;
           }
           if (sample < 3) {
-            pedestalPreSample += eeADCCounts[sample];
             pedestalPreSampleAnalog +=
                 eeADCCounts[sample] * ECalgainConv_[(int)eeADCGains[sample]] * ECalbarrelADCtoGeV_;
           }
         }
-        pedestalPreSample /= 3.;
         pedestalPreSampleAnalog /= 3.;
 
         // calculate pedestal subtracted digi energy in the crystal

--- a/Validation/GlobalDigis/src/GlobalDigisProducer.cc
+++ b/Validation/GlobalDigis/src/GlobalDigisProducer.cc
@@ -325,7 +325,6 @@ void GlobalDigisProducer::fillECal(edm::Event &iEvent, const edm::EventSetup &iS
 
       double Emax = 0;
       int Pmax = 0;
-      double pedestalPreSample = 0.;
       double pedestalPreSampleAnalog = 0.;
 
       for (int sample = 0; sample < nrSamples; ++sample) {
@@ -348,11 +347,9 @@ void GlobalDigisProducer::fillECal(edm::Event &iEvent, const edm::EventSetup &iS
           Pmax = sample;
         }
         if (sample < 3) {
-          pedestalPreSample += ebADCCounts[sample];
           pedestalPreSampleAnalog += ebADCCounts[sample] * ECalgainConv_[(int)ebADCGains[sample]] * ECalbarrelADCtoGeV_;
         }
       }
-      pedestalPreSample /= 3.;
       pedestalPreSampleAnalog /= 3.;
 
       // calculate pedestal subtracted digi energy in the crystal
@@ -430,7 +427,6 @@ void GlobalDigisProducer::fillECal(edm::Event &iEvent, const edm::EventSetup &iS
 
       double Emax = 0;
       int Pmax = 0;
-      double pedestalPreSample = 0.;
       double pedestalPreSampleAnalog = 0.;
 
       for (int sample = 0; sample < nrSamples; ++sample) {
@@ -454,11 +450,9 @@ void GlobalDigisProducer::fillECal(edm::Event &iEvent, const edm::EventSetup &iS
           Pmax = sample;
         }
         if (sample < 3) {
-          pedestalPreSample += eeADCCounts[sample];
           pedestalPreSampleAnalog += eeADCCounts[sample] * ECalgainConv_[(int)eeADCGains[sample]] * ECalbarrelADCtoGeV_;
         }
       }
-      pedestalPreSample /= 3.;
       pedestalPreSampleAnalog /= 3.;
 
       // calculate pedestal subtracted digi energy in the crystal

--- a/Validation/RecoParticleFlow/plugins/PFAnalysisNtuplizer.cc
+++ b/Validation/RecoParticleFlow/plugins/PFAnalysisNtuplizer.cc
@@ -1176,17 +1176,7 @@ void PFAnalysis::associateClusterToSimCluster(const vector<ElementWithIndex>& al
   for (const auto& detids : detids_elements) {
     int isimcluster = 0;
     if (!detids.empty()) {
-      double sum_e_tot = 0.0;
-      for (const auto& c : detids) {
-        sum_e_tot += c.second;
-      }
-
       for (const auto& simcluster_detids : simcluster_detids_) {
-        double sum_e_tot_sc = 0.0;
-        for (const auto& c : simcluster_detids) {
-          sum_e_tot_sc += c.second;
-        }
-
         //get the energy of the simcluster hits that matches detids of the rechits
         double cmp = detid_compare(detids, simcluster_detids);
         if (cmp > 0) {

